soc/nordic/common/ironside/call.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nrf/ironside/call.h>
+#include <zephyr/cache.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/mbox.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/barrier.h>
+#include <zephyr/sys/math_extras.h>
+#include <zephyr/sys/util.h>
+
+#define DT_DRV_COMPAT nordic_ipc_ironside
+
+#define SHM_NODE     DT_INST_PHANDLE(0, memory_region)
+#define NUM_BUFS     (DT_REG_SIZE(SHM_NODE) / sizeof(struct ironside_call_buf))
+#define ALL_BUF_BITS BIT_MASK(NUM_BUFS)
+
+/* Note: this area is already zero-initialized at reset time. */
```

Comment:
Is the boot report not placed in here?

Reply:
The boot report is in a different place. Have a look at the memory map.